en/config/actuators.md

```diff
@@ -140,32 +140,60 @@ Note that you will need to also ensure that the ESC associated with bidirectiona
 The control surfaces section of the geometry panel lets you set the number and types of control surfaces that are present on the vehicle.
 You may also need to set trim and slew rate values in some cases.
 More advanced users can also configure the roll scale, yaw scale, and pitch scale (generally the defaults are acceptable, and this is not needed).
-
 An "example" control surface section for a vehicle with two ailerons is shown below.
 Note that ailerons only affect roll, so the pitch and yaw fields are disabled.
 
 ![Control Surface Setup Example](../../assets/config/actuators/control_surfaces_geometry.png)
 
+:::note
+Only the most common settings are displayed by default.
+Select the **Advanced** checkbox in the top right corner of the view to display all settings.
+:::
+
 The fields are:
 
 - `Control Surfaces`: The number of control surfaces (set this first!)
 - `Type`: The type of each control surface: `LeftAileron`, `RightAileron`, `Elevator`, `Rudder`, `Left Elevon`, `Right Elevon`, `Left V-Tail`, `Right V-Tail`, `Left Flap`, `Right Flap`, `Airbrakes`, `Custom`.
-- `Roll scale`: Effectiveness of actuator around roll axis (normalised: -1 to 1).
+- `Roll Torque`: Effectiveness of actuator around roll axis (normalised: -1 to 1).
```

Comment:
I changed these from scale to Torque, as that is what I am now seeing in QGC and your image below. Is the definition still correct?

Reply:
yeah, "roll torque scale" would be better but too long, but I think people understand what it means with the help of the docs here. And ofc being consistent is the most important thing.